depends/packages/twisted.mk

```diff
@@ -0,0 +1,10 @@
+package=twisted
+$(package)_version=15.0.0
+$(package)_download_path=https://pypi.python.org/packages/source/T/Twisted
+$(package)_file_name=Twisted-$($(package)_version).tar.bz2
+$(package)_sha256_hash=ac609262253057cf2aeb9dc049ba7877d646f31b4caef06a50189a023df46b51
+$(package)_dependencies=native_python2 native_zope.interface
+
+define $(package)_stage_cmds
+  $($(package)_prefix)/native/bin/python setup.py install --prefix=$($(package)_staging_prefix_dir)
```

Comment:
Thanks. Those examples appear to work as is. And I can do something similar to the psutil example with twisted.

I don't understand the psutil example and your use of $(package)_config_env. Isn't $(package)_config_cmds set to default to do nothing. And isn't $(package)_config_env only used in conjunction with the $(package)_config_cmds stage? So I don't get how $(package)_config_env is making a difference without a $(package)_config_cmds definition.